src/modules/objectsearch/__tests__/__snapshots__/ObjectSearchComponent-test.js.snap

```diff
@@ -35,7 +35,7 @@ exports[`ObjectSearchComponent should display object 1 1`] = `
               placeholder=" "
               style={Object {}}
               type="text"
-              value="1234" />
+              value="" />
```

Comment:
fix  test, dont fix snapshot :) see comment above in ObjectSearchComponent-test.js.